entries/m/myfritz.net.json

```diff
@@ -0,0 +1,20 @@
+{
+  "MyFRITZ!Net": {
+    "domain": "myfritz.net",
+    "url": "https://www.myfritz.net",
+    "additional-domains": [
+      "sso.myfritz.net"
+    ],
+    "tfa": [
+      "totp",
+      "custom-hardware"
+    ],
+    "custom-hardware": [
+      "Smartphone biometrics (fingerprint, face recognition)"
+    ],
```

Comment:
How are the biometrics read? Is it through an app that they have on the Play Store and/or App Store?

Reply:
I just rechecked that - it seems they only use Webauthn to support hardware tokens as well as smartphone biometrics.